### PR TITLE
Separate UI-related end-user docs from commands section

### PR DESCRIPTION
**Pull request by mickmister**
#### Summary

Currently the [Using /jira commands](https://mattermost.gitbook.io/plugin-jira/end-user-guide/using-jira-commands) page has information about using the modals shuffled into the command explanations. This PR separates the two sections into different pages.

### CODE REVIEW

**Review by jfrerich — APPROVED** [review]
LGTM! I noticed the Jira Groups image was removed.  Is there a section of the docs that uses and discusses Jira Group permissions?

**Comment by mickmister** [comment]
@justinegeffen Do these docs-related PRs need to be cherrypicked into the docs branch, or should they just go into the docs branch in the first place?

File: docs/end-user-guide/using-jira-commands.md [hunk 1]
```diff
@@ -1,60 +1,34 @@
...
-* **Single-Choice Issue:** Custom fields, and built-in fields such as **Issue Type** and **Priority**. 
-* **Assignee:** System field.
-
-### Attach Messages to Jira issues
+**Note:** For the `/jira instance settings` command, [setting] can be `notifications` and [value] can be `on` or `off`
 
-Keep all information in one place by attaching parts of Mattermost conversations in Jira issues as comments. To attach a message, hover over the relevant message and select **\(...\) > More Actions > Attach to Jira Issue**.
+### Authenticating with Jira
 
-![You can attach a message to an existing Jira ticket](../.gitbook/assets/image%20%286%29.png)
```
Comment (jfrerich) [inline docs/end-user-guide/using-jira-commands.md:51]: @mickmister, removing this image is what I was referring to.  Is it referenced anywhere else or can the file be deleted?

Comment (mickmister) [reply docs/end-user-guide/using-jira-commands.md:51]: @jfrerich That linked image was moved to another file, `docs/end-user-guide/interacting-with-forms.md`. Notice how the original page is called `using-jira-commands`, which has nothing to do with UI features of the plugin, including the "Attach to Issue" feature, hence this PR being created 🙂 

Take a look at https://mattermost.gitbook.io/plugin-jira/end-user-guide/using-jira-commands and notice how the first section is about commands but then has UI-related info shuffled within the text. Also the actual image doesn't match the Attach feature, hence this PR https://github.com/mattermost/mattermost-plugin-jira/pull/782

**Comment by mickmister** [comment]
@justinegeffen I think is ready to merge, though I'm not sure what branch this should be targeting. Should we avoid using master for the docs PRs?

**Comment by mickmister** [comment]
@justinegeffen After merging this PR, the docs don't show the newly added `interacting-with-forms` page. Do you know why this might be the case?

**Pull request merged**